pkg/providers/json/json.go

```diff
@@ -0,0 +1,137 @@
+package json
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/jexia/semaphore/pkg/broker"
+	"github.com/jexia/semaphore/pkg/broker/logger"
+	"github.com/jexia/semaphore/pkg/providers"
+	"github.com/jexia/semaphore/pkg/specs"
+	"go.uber.org/zap"
+)
+
+type collection struct {
+	Flows     specs.FlowList     `json:"flows,omitempty"`
+	Proxy     specs.ProxyList    `json:"proxy,omitempty"`
+	Endpoints specs.EndpointList `json:"endpoints,omitempty"`
+	Services  specs.ServiceList  `json:"services,omitempty"`
+	Schemas   specs.Schemas      `json:"schemas,omitempty"`
+}
+
+func (collection *collection) Append(incoming collection) {
+	collection.Flows.Append(incoming.Flows)
```

Comment:
not sure it is going to work (since `Append` method has a pointer receiver)
please, correct me if I am wrong